templates/contrib/pc/README.md

```diff
@@ -0,0 +1,14 @@
+# Overview 
+
+This directory provides a RHEL6 based meta-template, to support legacy guest OS
```

Comment:
I think you will have to change this description to something like:
1) create copy of rhel 6 template
2) update machine type on the new template
because if you change the something on original template, ssp operator will replace it back to its original state.

Reply:
hmm, I think I'm not following your thought...
Isn't creating a separate meta-template address this issue?

Reply:
Sorry my fault, I wrongly understand the command bellow